web/modules/custom/asset_distribution/asset_distribution.behat.inc

```diff
@@ -299,4 +299,46 @@ class AssetDistributionSubContext extends DrupalSubContextBase implements Drupal
     return $fields;
   }
 
+  /**
+   * Checks if a distribution in compact view contains download links.
+   *
+   * @param string $distribution
+   *    The name of the distribution.
+   *
+   * @Then I should see the download link in the :distribution( asset) distribution
+   *
+   * @throws \Exception
```

Comment:
We should respect the order of the parameters as outlined in the coding standards, even though this is not checked by Coder at the moment.